README.md

```diff
@@ -1,2 +1,12 @@
 # comp110-worksheet-5
 Base repository for COMP110 worksheet 5
+
+(a)	Finds duplicates within a list.
+(b)	Checks itself (n), then checks itself again (n * n  n^2).
+(c)	The algorithm still checks every number against every element behind it. 
+(d)	The first algorithm checks every element twice, whereas the second algorithm that checks each element behind itself, only checks each element once.
+(e)	Yes, because for every element checked in the list, every other element is compared, meaning O(n^2)
+(f)	O(n log n)
+(g)	O(log n)
+(h)	Algorithm 2, as it takes half the amount of time., and doesn’t check each element many times.
+(i) In a list less than 100 elements, the "slower" algorithm actually completes faster.
```

Comment:
Not necessarily 100, but certainly for some value of n (implementation dependent)